_rules/role-attribute-valid-value-674b10.md

```diff
@@ -42,19 +42,19 @@ Each test target has at least one token which is a valid value corresponding to
 
 ## Assumptions
 
-- This rule assumes that the `role` attribute is used to provide an ARIA [semantic role][] to the elements. If it is used for other purposes, this rule shouldn't be used.
-- This rule assumes that elements with a `role` attribute have their intended structure and relationship conveyed through some sort of presentation. If it is not the case, it is possible to fail this rule while still satisfying [Success Criterion 1.3.1 Info and Relationship][sc131].
-- This rule assumes that the intended role of the element is not its [implicit role][]. If no token is valid, User Agents will default to the [implicit role][] for the element; if that role is the intended one, it is possible to fail this rule but still satisfy [Success Criterion 1.3.1 Info and Relationships][sc131].
+There are no assumptions.
```

Comment:
```suggestion
This rule assumes that the `role` attribute is used to provide an ARIA [semantic role][] to the elements. If it is used for other purposes, this rule shouldn't be used.
```

Hey @WilcoFiers , as per our call, it's worth leaving the first paragraph in as the rule is mapping to the technique and there might be, however small, possibility of using the `role` attribute intentionally not for accessibility.